lib/GLPI/Agent/Task/Inventory/Linux/AntiVirus/Sentinelone.pm

```diff
@@ -0,0 +1,71 @@
+package GLPI::Agent::Task::Inventory::Linux::AntiVirus::Sentinelone;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use UNIVERSAL::require;
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    return canRun('/opt/sentinelone/bin/sentinelctl');
+}
+
+sub doInventory {
+    my (%params) = @_;
+
+    my $inventory = $params{inventory};
+    my $logger    = $params{logger};
+
+    my $antivirus = _getSentineloneInfo(logger => $logger);
+    if ($antivirus) {
+        $inventory->addEntry(
+            section => 'ANTIVIRUS',
+            entry   => $antivirus
+        );
+
+        $logger->debug2("Added $antivirus->{NAME}" . ($antivirus->{VERSION} ? " v$antivirus->{VERSION}" : ""))
+            if $logger;
+    }
+}
+
+sub _getSentineloneInfo {
+	my @command1 = getAllLines(command => '/opt/sentinelone/bin/sentinelctl version');
+	my @command2 = getAllLines(command => '/opt/sentinelone/bin/sentinelctl engines status');
+	my @command3 = getAllLines(command => '/opt/sentinelone/bin/sentinelctl control status');
+	my @command4 = getAllLines(command => '/opt/sentinelone/bin/sentinelctl management status');
+
+    my @output = (@command1,@command2,@command3,@command4)
+        or return;
```

Comment:
I see here one problem and a cleaner way to prepare `@output`. The problem is not critical. You just forgot to get `%params` and include it in `getAllLines()` calls so the run command is logged. Also it's better to return on first error and `push` gotten lines into the still declared `@output` array as we got them.

It seems also you're using tabulation for indentation. Please, can you reformat your source to use 4 spaces indentation ?

```suggestion
sub _getSentineloneInfo {
    my (%params) = @_;

    my @output = getAllLines(command => '/opt/sentinelone/bin/sentinelctl version', %params)
        or return;
    push @output, getAllLines(command => '/opt/sentinelone/bin/sentinelctl engines status', %params)
        or return;
    push @output, getAllLines(command => '/opt/sentinelone/bin/sentinelctl control status', %params)
        or return;
    push @output, getAllLines(command => '/opt/sentinelone/bin/sentinelctl management status', %params)
        or return;
```

Reply:
Indeed I've another suggestion:
```suggestion
sub _getSentineloneInfo {
    my (%params) = @_;

    my $cmd = '/opt/sentinelone/bin/sentinelctl';

    my @output = getAllLines(
        command => "$cmd version && $cmd engines status && $cmd control status && $cmd management status",
        %params
    )
        or return;
```
Can you try this one is giving the expected result ? It should and it looks indeed better.